packages/ffe-datepicker-react/src/calendar/Header.js

```diff
@@ -1,50 +1,139 @@
+import i18n from '../i18n/i18n';
 
 export default function Header(props) {
     const {
         datepickerId,
         month,
+        year,
+        language,
         nextMonthHandler,
-        nextMonthLabel,
         previousMonthHandler,
-        previousMonthLabel,
-        year,
+        todayHandler,
+        selectYearHandler,
     } = props;
 
+    function renderYearOptions() {
+        const distribution = 5;
+        const minYear = year - distribution;
+        const maxYear = year + distribution;
+        const options = [];
+        let currentYear = minYear;
+        let option;
+
+        while (currentYear <= maxYear) {
+            option = (
+                <option key={currentYear} value={currentYear}>
+                    {currentYear}
+                </option>
+            );
+            options.push(option);
+            currentYear += 1;
+        }
+
+        return options;
+    }
+
+    const todayLabel = i18n[language].TODAY;
+
     return (
-        <div className="ffe-calendar__header">
-            <div className="ffe-calendar__header-inner-wrapper">
-                <button
-                    className="ffe-calendar__month-nav ffe-calendar__previous"
-                    onClick={previousMonthHandler}
-                    tabIndex="-1"
-                    title={previousMonthLabel}
-                    type="button"
-                >
-                    <ChevronIkon className="ffe-calendar__icon-prev" />
-                </button>
-                <header
-                    aria-live="assertive"
-                    aria-atomic="true"
-                    className="ffe-calendar__title"
-                    id={`${datepickerId}-title`}
-                >
-                    <div id={`${datepickerId}__month-label`}>
-                        <span className="ffe-calendar__month">{month}</span>
-                        <span className="ffe-calendar__year">{year}</span>
+        <div id={`${datepickerId}-header`} className="ffe-calendar__header">
+            <div
+                id={`${datepickerId}-settings`}
+                className="ffe-calendar__header-container"
+            >
+                <div className="ffe-calendar__header-wrapper">
+                    <div className="ffe-calendar__header-define" />
+                    <div className="ffe-calendar__header-break ffe-calendar__header-break-1" />
+                    <div className="ffe-calendar__header-block" />
+                    <div className="ffe-calendar__header-define" />
+                    <div className="ffe-calendar__header-break ffe-calendar__header-break-2" />
+                    <div className="ffe-calendar__header-block" />
+
+                    <div
+                        id={`${datepickerId}-view1`}
+                        className="ffe-calendar__header-content"
+                    >
+                        <div className="ffe-calendar__header-inner-1">
+                            <div>
```

Comment:
mye divar run elementerna. Er egentligen ikke feil, men trengs de? Blir de visuell forskjell med de?

Reply:
There’s definitely room for improvement in the markup, but I guess I made it this way to not overcomplicate the less file.

Reply:
Just a question if the could be removed without breaking the style.